qldbdriver/logger.go

```diff
@@ -58,18 +58,23 @@ func (qldbLogger *qldbLogger) logf(verbosityLevel LogLevel, message string, args
 	if verbosityLevel <= qldbLogger.verbosity {
 		switch verbosityLevel {
 		case LogInfo:
-			qldbLogger.logger.Log(fmt.Sprintf("[INFO] "+message, args...))
+			qldbLogger.logger.Log(fmt.Sprintf("[INFO] "+message, args...), verbosityLevel)
 		case LogDebug:
-			qldbLogger.logger.Log(fmt.Sprintf("[DEBUG] "+message, args...))
+			qldbLogger.logger.Log(fmt.Sprintf("[DEBUG] "+message, args...), verbosityLevel)
 		default:
-			qldbLogger.logger.Log(fmt.Sprintf(message, args...))
+			qldbLogger.logger.Log(fmt.Sprintf(message, args...), verbosityLevel)
 		}
 	}
 }
 
-type defaultLogger struct{}
+type defaultLogger struct {
+	// The verbosity level of the logs that the logger should receive.
+	verbosity LogLevel
+}
 
 // Log the message using the built-in Golang logging package.
-func (logger defaultLogger) Log(message string) {
-	log.Println(message)
+func (logger *defaultLogger) Log(message string, verbosity LogLevel) {
+	if verbosity <= logger.verbosity {
```

Comment:
The logger implementation itself is NOT what controls what comes through. There should be no changes here.

Reply:
Addressed.